lib/checksum.js

```diff
@@ -34,14 +47,16 @@ const utils = require('./utils');
  * @param {Object} options - options
  * @param {Number} options.imageSize - image size in bytes
  * @param {Number} options.chunkSize - chunk size
+ * @param {String[]} options.algorithms=['crc32'] - checksum hash algorithms
```

Comment:
What is the reasoning behind computing multiple checksum types? I guess we can go with only one the user asks for, or do the user's one + crc32 (so we only show crc32 on the GUI for instance).